core/Files.cc

```diff
@@ -298,4 +304,24 @@ string File::censorFilePathForSnapshotTests(string_view orig) {
     }
 }
 
+bool File::isPackagedTest() const {
+    return flags.isPackagedTest;
+}
+
+bool File::hasParseErrors() const {
+    return flags.hasParseErrors;
+}
+
+void File::setHasParseErrors(bool value) {
+    flags.hasParseErrors = value;
+}
+
+bool File::cached() const {
+    return flags.cached;
+}
+
+void File::setCached(bool value) {
+    flags.cached = value;
+}
```

Comment:
slight preference for just exposing the flags directly, rather than creating these wrapper functions